src/main/scala/org/broadinstitute/clio/ClioServer.scala

```diff
@@ -32,7 +33,7 @@ object ClioServer
   private val wrapperDirectives: Directive0 = {
     auditRequest & auditResult & completeWithInternalErrorJson & auditException & mapRejectionsToJson
   }
-  private val innerRoutes: Route = concat(statusRoutes)
+  private val innerRoutes: Route = concat(authorizationRoute, statusRoutes)
```

Comment:
This adds `/authorization` as new endpoint but it doesn't make authorization required by any other APIs, correct?